### DISCUSSION TITLE
Refresh Rate choices

### DISCUSSION THREAD

**Discussion by Valour549** [seerge/g-helper#3433] [Q&A]
When using Standard Mode, the available refresh rate options were 60Hz and 240Hz in Windows settings. Now I switched to Ultimate mode, there are a lot more options. When I tried Dynamic and went to the [UFO test](https://www.testufo.com/) it showed 120 FPS, do you know how Dynamic work exactly and is it compatible with G-Helper?
![image](https://github.com/user-attachments/assets/ef06ff9e-e05e-4e8e-9192-a86dcf37a4c2)

**Comment by seerge** [comment]
@Valour549 hello,

I think Dynamic Refresh rate is something that is managed by GPU drivers directly if both GPU and display support it.

G-Helper uses official windows API to set refresh rates (as this is the only way to do it), and dynamic is not supported there for setting.

But as far as you don’t have `Auto` Display mode set in the app - it won’t interfere with refresh rates changes by itself, so you can use Dynamic in windows if it works for you:)

**Reply by Valour549** [reply]
Got it.

BTW I noticed a small bug with Visual Mode. So last time the Disabled option got added, and when it's selected and you change the color-temperature nothing happens (which is expected). But when you select a Gamut, the Visual Mode actually switches back on to the last visual mode used, but the selection is still stuck on Disabled. It should either do nothing, or update to the correct visual mode. I personally think it should do nothing so its behavior is consistent with that of the color-temperature.

Or better yet, disable color-temperature and gamut selection when Disabled is selected. It will remind the user to actually select the mode they want.
